api/src/main/java/org/pmiops/workbench/rdr/RdrExportServiceImpl.java

```diff
@@ -193,11 +193,6 @@ private RdrResearcher toRdrResearcher(DbUser dbUser) {
             RdrExportEnums.sexAtBirthToRdrSexAtBirth(
                 dbDemographicSurvey.getSexAtBirthEnum().get(0)));
       }
-      if (dbDemographicSurvey.getSexualOrientationEnum() != null) {
```

Comment:
Is it an AC to not export this anymore?

Reply:
We aren't capturing this data in enum field anymore. I think there's a conversation with the RDR team about importing the new fields that @NehaBroad was leading.